netlify.toml

```diff
@@ -4,8 +4,7 @@
   framework = "#custom"
 
 [context.deploy-preview]
-  # only do a preview deply when there is a website change
-  ignore = "git diff --quiet main...HEAD -- packages/paste-website packages/paste-theme-designer ':!packages/**/__tests__/*' ':!packages/paste-website/stories'"
+  ignore = "git diff --quiet main...HEAD -- packages/paste-core packages/paste-website packages/paste-theme-designer ':!packages/**/__tests__/*' ':!packages/paste-website/stories'"
```

Comment:
question: what does adding paste-core do here?

Reply:
This includes changes under paste-core as things that will trigger a preview website build - the builds were being skipped, but I do believe we would want core changes to trigger preview builds.